client/settings/general-settings-section/payment-method-description.js

```diff
@@ -3,6 +3,7 @@
  */
 import React from 'react';
 import styled from '@emotion/styled';
+import PaymentMethodSetupHelp from './payment-method-setup-help';
```

Comment:
This should be prefaced with an Internal dependencies docblock ^

Reply:
Ah thanks, fixed in a08b9b2b75ecd650d1f12f1a81e8501ea3b9fbb8 - FYI we're removing that rule in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1854#discussion_r709661064

Reply:
🥳 I'm not a fan myself